you-have-mail-android/app/build.gradle

```diff
@@ -18,7 +18,7 @@ android {
 
     defaultConfig {
         applicationId "dev.lbeernaert.youhavemail"
-        minSdk 29
+        minSdk 26
```

Comment:
What are the implication of lowering this target?

Reply:
I just needed it to enable this application to run on some of my older smartphones with Android 8 Oreo (Sdk 26)
But minSdk can be reverted back to 29. You are the creator, you can decide

Reply:
Was everything working correctly on those devices? I don't have a way to test that currently.

Reply:
I have a smartphone Sony Xperia with Android 8 Oreo (Sdk 26). Yes, this application works correctly on it. Reliably shows notifications and allows entering Proton Mail application, nice :)